src/java.base/share/classes/jdk/internal/classfile/impl/BootstrapMethodEntryImpl.java

```diff
@@ -76,9 +76,7 @@ public boolean equals(Object obj) {
 
     static int computeHashCode(MethodHandleEntryImpl handle,
                                List<? extends LoadableConstantEntry> arguments) {
-        int hash = handle.hashCode();
-        hash = 31 * hash + arguments.hashCode();
-        return AbstractPoolEntry.phiMix(hash);
+        return 31 * handle.hashCode() + arguments.hashCode();
```

Comment:
Should we update the algorithm here, as this algorithm processes hash codes of the 2nd last item of arguments and the handle the same, both just multiplied by 31?

Also need a copyright year update.

Reply:
Right, BSMEntries are also stored in EntryMap.